packages/babel-helper-replace-supers/src/index.js

```diff
@@ -263,7 +263,7 @@ export default class ReplaceSupers {
       property = node.property;
       computed = node.computed;
     } else if (t.isUpdateExpression(node) && isMemberExpressionSuper(node.argument)) {
-      const binary = t.binaryExpression(node.operator[0], node.argument, t.numericLiteral(1));
+      const binary = t.assignmentExpression(node.operator[0] + "=", node.argument, t.numericLiteral(1));
```

Comment:
The other calls to specHandleAssignmentExpression take in an AssignmentExpression, not a BinaryExpression. Since the function currently just takes the first character of the operator, it still worked with a BinaryExpression, but the new implementation that strips off the last character freaked out when it got a BinaryExpression with "+" as the operator.